### ISSUE TITLE
Per-organizer logging

### ISSUE THREAD

**Issue by ramontayag** [adomokos/light-service#159]
@adomokos, what are you thoughts on per-organizer logging?

I find logging to be helpful, especially on a newly deployed organizer to see what problems may occur. However, since LightService is used extensively, the logs have become too verbose, logging about other organizers/actions that I no longer care to see in detail.

**Comment by adomokos** [comment]
I like it! Care to start a PR with it?

**Comment by ramontayag** [comment]
Yup, I am happy to do it, but I will be able to start in about a week or two. :)

**Comment by adomokos** [comment]
Great! Maybe submit a PR early on,  even with incomplete code, to make sure you are going down the right path. Thanks for it in advance!

**Comment by adomokos** [comment]
I'll release it shortly, I just want to sneak in another deprecation warning for key aliases.
